react/graphql/getOrderGroup.graphql

```diff
@@ -30,6 +30,28 @@ query getOrderGroup($orderGroup: String) {
           listPrice
           isGift
           quantity
+          attachments {
+            content {
+              text
```

Comment:
Attachment content have arbitrary keys and values

Reply:
See https://github.com/vtex/checkout-confirmation-ui/blob/master/src/mock/mockAttachments.json#L55

Reply:
That is so weird 😅
But I updated the query according to the adjustment make in PR #5 from `vtex/order-placed-graphql`